sqlitecluster/SQLiteNode.cpp

```diff
@@ -2557,11 +2531,12 @@ bool SQLiteNode::hasQuorum() const {
 }
 
 void SQLiteNode::prePoll(fd_map& fdm) const {
```

Comment:
Can you please summarize what changes to this method are functional vs refactoring? There's a lot of change so it's a bit hard to follow

Reply:
It should have the same effect as before, but no longer needs direct access to the peer's socket, which is now private.